crmsh/utils.py

```diff
@@ -2845,7 +2845,15 @@ def is_dev_used_for_lvm(dev, peer=None):
     """
     Check if device is LV
     """
-    return get_dev_info(dev, "TYPE", peer=peer) == "lvm"
+    return "lvm" in get_dev_info(dev, "TYPE", peer=peer)
+
+
+def is_dev_used_for_device_mapper(dev, peer=None):
+    """
+    Check if device is a device mapper
+    """
+    out = get_dev_info(dev, "TYPE", peer=peer)
+    return re.search("(raid|lvm)", out) is not None
```

Comment:
>What about if 'dev' not exist yet, eg. /dev/md127?

When calling `is_dev_used_for_device_mapper`, we could make sure the `dev` already exist on init node
Since on this time init node was running cluster service

> Well, seem a simpler and more strict approach even no need the above call. How about,
return re.search("(disk|part)", out)

OK, seems more strict, thanks!

Reply:
`return re.search("(disk|part)", out) is None` is more clear

Reply:
> > What about if 'dev' not exist yet, eg. /dev/md127?
> 
> When calling `is_dev_used_for_device_mapper`, we could make sure the `dev` already exist on init node
> Since on this time init node was running cluster service

Maybe some typo. The joint node is problem maker. We can not assume `dev` exist on the join node, indeed. Well, it does not impact to the current code proposal yet anyway.

> 
> > Well, seem a simpler and more strict approach even no need the above call. How about,
> > return re.search("(disk|part)", out)
> 
> OK, seems more strict, thanks!

I prefer to change the function name to fit the whole OS terminology better. How about,
```
is_dev_a_plain_raw_disk_or_partition():
    return re.search("(disk|part)", out) is not None
```

And reverse the caller condition accordingly, 
```
if utils.is_dev_a_plain_raw_disk_or_partition(target, peer):
```

Reply:
Ok, changed as `is_dev_a_plain_raw_disk_or_partition`